main.go

```diff
@@ -245,6 +249,9 @@ func main() {
 		"Threshold which safemode checks against to see if the number of targets is over safety measures within a cluster")
 	handleFatalError(viper.BindPFlag("controller.safemode.clusterThreshold", pflag.Lookup("safemode-cluster-threshold")))
 
+	pflag.StringVar(&cfg.Controller.CloudProviders.Aws.IPRangesURL, "cloud-providers-aws-ip-ranges-url", "", "URL used to pull ip ranges from AWS (defaulted to \"\")")
```

Comment:
We default to an empty string, but won't we error out in our Cloud Service if that happens? How do we want to handle a situation where we don't get a config (or if the config we do have doesn't work)? Fail all disruptions that try to block to aws?

Reply:
Changed this part to not have any configuration as the URL similar for everyone.
For now, we don't start the controller if an error is thrown at the parsing of the ip ranges.